shared/js/background/classes/ad-click-attribution-policy.js

```diff
@@ -134,9 +157,26 @@ export class AdClick {
         restoredAdClick.adClickRedirect = adClick.adClickRedirect
         restoredAdClick.expires = adClick.expires
         restoredAdClick.clickExpires = adClick.clickExpires
+        restoredAdClick.adClickDNR = adClick.adClickDNR
```

Comment:
Doesn't this need to be constructed again?
If you kill the worker, is this restoring properly?

Reply:
At this point all we need is the rule ID. Reconstructing also generates a new rule id and a new DNR, which we don't want. 
We could recreate it here, but then we also need to explicitly call some getID and createRule methods.